perfkitbenchmarker/providers/openstack/os_network.py

```diff
@@ -27,134 +29,143 @@
 
 
 class OpenStackFirewall(network.BaseFirewall):
+  """
+  An object representing OpenStack Firewall based on Secure Groups.
+  """
+
+  CLOUD = providers.OPENSTACK
+
+  def __init__(self):
+    self._lock = threading.Lock()  # Guards security-group rule set
+    self.sec_group_rules_set = set()
+
+    with self._lock:
+      cmd = utils.OpenStackCLICommand(self, 'security group', 'show',
```

Comment:
Please move hard-coded constants (`'security group'`, `'perfkit_sc_group'`) to module-level constants at the top of the file.